### ISSUE TITLE
output data types - floats vs ints

### ISSUE THREAD

**Issue by erialC-P** [GeoscienceAustralia/dea-intertidal#1]
test the suitability of using ints instead of floats as outputs from:
- NIDEM (cm instead of m)
- exposure (% instead of decimals - need to determine whether loss of spatial resolution is detrimental to the product)
- bias/offset (as above)

**Comment by erialC-P** [comment]
tested ds.idxmin.astype('int') on exposure array but didn't work as expected. Need to return to this down the track.

**Comment by robbibt** [comment]
As in here? 
[    dem['idxmin'] = dem['diff'].idxmin(dim='quantile')](https://github.com/GeoscienceAustralia/dea-intertidal/blob/c59fc4bebef9bef4304f9f04b83a3d364319051a/intertidal/exposure.py#L39)

Does this work?
```
import numpy as np
dem['diff'].idxmin(dim='quantile').astype(np.int16)
```

**Comment by erialC-P** [comment]
Afraid not. The array values all change from being between 0 and 1 to between 0 +- 0.05
The xarray.astype doco indicates there might be some more thinking to do on the conversion…

Dr Claire Phillips (she/her)  |  Remote Sensing Scientist
Digital Earth Australia  |  Space Division
Geoscience Australia
t +61 2 6249 9462    ga.gov.au<https://www.ga.gov.au/>

**Comment by erialC-P** [comment]
> Afraid not. The array values all change from being between 0 and 1 to between 0 +- 0.05 The xarray.DataArray.astype doco indicates there might be some more thinking to do on the conversion…

Update: seems to work if values (on the extents dataset) are scaled up by multiplying by 100 before applying astype func.

**Comment by erialC-P** [comment]
New TODO
With respect to the exposure layer: all former nan values were given a value of 0 when dtype changed from float to int. This may not make sense with the purpose of the exposure layer. TODO: check how floats of 0 compare to int 0's in exposure outputs

**Comment by robbibt** [comment]
Closed by #13
